jbpm-designer-backend/src/main/java/org/jbpm/designer/repository/vfs/RepositoryDescriptorProvider.java

```diff
@@ -67,10 +67,8 @@ public RepositoryDescriptor getRepositoryDescriptor(Space space,
             branchName = "master";
         }
 
-        if (knownRepositories.containsKey(branchName + "@" + repositoryAlias)) {
-            return knownRepositories.get(branchName + "@" + repositoryAlias);
-        } else if (knownRepositories.size() == 1) {
```

Comment:
this is ok to remove right?

Reply:
@ederign This was causing the AF-1061 issue. I don't see a reason to keep it after the project oriented changes, since doesn't make sense to return any repository description just because it is the only one on this cache. The `else` clause after that should suffice.